app/services/hyrax/works_count_service.rb

```diff
@@ -10,6 +11,7 @@ def search_results_with_work_count(access)
       works = search_results(access)
 
       works.map do |work|
+        next if work['system_create_dtsi'].nil?
```

Comment:
I don't understand what the purpose of this is. In what situation would a record not have this? Can you document that? Is there a test for this?

Reply:
Honestly, I'm not sure it does need it. I was getting periodic build failures rais 5.0.6 for this:
```
 1) The homepage as an admin shows featured works that I can sort
     Failure/Error: created_date = DateTime.parse(work['system_create_dtsi']).in_time_zone.strftime("%Y-%m-%d")
     
     TypeError:
       no implicit conversion of nil into String
     # ./app/services/hyrax/works_count_service.rb:14:in `parse'
     # ./app/services/hyrax/works_count_service.rb:14:in `block in search_results_with_work_count'
     # ./app/services/hyrax/works_count_service.rb:13:in `map'
     # ./app/services/hyrax/works_count_service.rb:13:in `search_results_with_work_count'
```
As far as I know the homepage should never hit that code.

Reply:
This seems like a good change, but it also seems like it should be it's own pull request. I'd also like to see this raise an error where it shows the whole document we're getting so we can do a better job of preventing this situation, rather than silently filtering unexpected values.